src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action=""
+      class="search"
+      data-qa="big">
```

Comment:
```suggestion
    <form action=""
      class="search"
      data-qa="big"
    >
```
Closing bracket should be on the new line